src/components/common/button.tsx

```diff
@@ -23,6 +23,10 @@ const StyledButton = styled.button<{ variant?: string }>`
         props.variant && props.variant === 'tertiary'
             ? `background-color: ${props.theme.componentsTheme.buttonTertiaryBackgroundColor};`
             : ''}
+    ${props =>
+        props.variant && props.variant === 'quaternary'
+            ? `background-color: ${props.theme.componentsTheme.buttonQuaternaryBackgroundColor};`
+            : ''}
```

Comment:
PP: Hi, could you use default props?, [here](https://www.styled-components.com/docs/advanced) there is an example